src/metabase/util.clj

```diff
@@ -13,7 +13,9 @@
             [flatland.ordered.map :refer [ordered-map]]
             [medley.core :as m]
             [metabase.config :as config]
+            [metabase.util.debugging-test :as dbg]
```

Comment:
you can't import `./test` namespaces into `./src` namespaces -- this might pass in unit tests but it will definitely fail when we go to build the uberjar. That file isn't going to be available

Reply:
You could try to `resolve` it at runtime however -- just keep in mind it will fail in prod